renovate.json

```diff
@@ -10,9 +10,27 @@
     }
   },
   "constraints": {
-    "python": "3.7"
+    "python": "~=3.7.0"
   },
-  "pip_requirements": {
-    "enabled": false
+  "prConcurrentLimit":5,
+  "stabilityDays":7,
+  "vulnerabilityAlerts":{
+     "labels":[
+       "type:security" 
+     ],
+     "stabilityDays":0
+  },
+  "labels": ["dependencies"],
+  "python":{
+    "addLabels": ["lang: python"]
+  },
+  "java":{
+    "addLabels": ["lang: java"]
+  },
+  "golang":{
+    "addLabels": ["lang: go"]
+  },
+  "nuget":{
+    "addLabels": ["lang: C#"]
```

Comment:
Suggestion: I looked at the existing labels. `lang: C#` is not one of them for some reason. 🤷  Maybe we'll just use `lang: dotnet`.

Reply:
that sounds good to me!